docs/example_go.md

```diff
@@ -3,29 +3,273 @@ title: "Example: Go"
 ```
 
-Each widget is implemented by a different microservice backend.
+The first line tells Tilt to build an image with the name `example-html-image`
+in the directory `.` (the current directory).
+
+The second line tells Tilt to load the Kubernetes
+[Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#creating-a-deployment)
+yaml. The image name in the `docker_build` call must match the container `image`
+reference in the `example-go` Deployment.
+
+The last line configures port-forwarding so that your server is
+reachable at http://localhost:8000/. The resource name in the `k8s_resource` call
+must match the Deployment's `metadata.name` in `kubernetes.yaml`.
+
+Try it! Run:
+
+```
+git clone https://github.com/windmilleng/tilt-example-go
+cd tilt-example-go/0-base
+tilt up
+```
+
+Tilt will open a browser showing the web UI, a unified view that shows you app
+status and logs. Your terminal will also turn into a status box if you'd like to
+watch your server come up there.
+
+When it's ready, you will see the status icon turn green. The logs in the
+botton pane will display "Serving files on port 8000."
+
+<figure>
+  <a class="is-image" href="https://cloud.tilt.dev/snapshot/AZzpx-YL_-WxDmRFwlk=">
+    <img src="assets/docimg/example-go-image-1.png">
+  </a>
+  <figcaption>The server is up! Click the screenshot to see an interactive snapshot.</figcaption>
+</figure>
+
+## Step 1: Let's Add Benchmark Trickery
+
+Before we try to make this faster, let's measure it.
+
+Tilt can run commands locally, so that you can integrate your existing
+scripts. We want a script that modifies Go code, so we can see how long it takes
+from a Go code change to a new process.
+
+In this example, we use [`local_resource`](local_resource.html), which lets you
+trigger local jobs, or run local servers. We add a `local_resource` to our
+[Tiltfile](https://github.com/windmilleng/tilt-example-go/blob/master/1-measured/Tiltfile)
+that records the start time in a Go file. We've also modified our server itself to
+read that start time and print the time elapsed.
+
+```python
+k8s_resource(
+    'example-go', 
+    port_forwards=8000, 
+    resource_deps=['deploy'])
+
+# Records the current time, then kicks off a server update.
+# Normally, you would let Tilt do deploys automatically, but this
+# shows you how to set up a custom workflow that measures it.
+local_resource(
+    'deploy',
+    './record-start-time.sh',
+)
+```
+
+The `local_resource()` call creates a local resource named `deploy`. The second
+argument is the script that it runs.
+
+Let's click the button on the `deploy` resource and see what happens!
+
+<figure>
+  <a class="is-image" href="https://cloud.tilt.dev/snapshot/AbakyOYLWWvW6J2BJFo=">
+    <img src="assets/docimg/example-static-html-image-2.png">
+  </a>
+  <figcaption>Step 1 complete. Click the screenshot to see an interactive snapshot.</figcaption>
+</figure>
+
+| Approach | Deploy Time |
```

Comment:
as per #465, this should be explicit html so we can style it (unless you reject the premise of #465 in which case 🤷‍♀ )

Reply:
haha yep. so add `{:.benchmark-report}` after the markdown tables instead pls